week5.1-ex1.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+
+def path_fun(path):
```

Comment:
Read this to understand how you should document a function
https://peps.python.org/pep-0257/